cmd/sdk-server/main.go

```diff
@@ -70,6 +71,14 @@ var (
 )
 
 func main() {
+	logLevelEnv := os.Getenv("LOG_LEVEL")
```

Comment:
Rather than use `os.Getenv` - please make this part of the `parseEnvFlags()` function and object that it returns - then we have all argument and environment variable parsing in a single place.

Reply:
Please ignore the above question. I've tested the latest code changes with the `Info`, `Error`, `Debug`, and `Warn` log levels. Please review the logs at https://gist.github.com/Kalaiselvi84/74fe77e6cce7ebc55485efbeb1e0d79d and share your feedback.

Reply:
Could you please confirm if the manual testing looks fine? Is this how we should be testing it?

Reply:
Yep! Looks perfect! 👍🏻